src/internal/operators/timeout.ts

```diff
@@ -336,52 +337,46 @@ export function timeout<T, R, M>(config: number | Date | TimeoutConfig<T, R, M>,
     // tell how many values we have seen so far.
     let seen = 0;
     const startTimer = (delay: number) => {
-      subscriber.add(
-        (timerSubscription = scheduler!.schedule(() => {
-          let withObservable: Observable<R>;
-          try {
-            withObservable = innerFrom(
-              _with!({
-                meta,
-                lastValue,
-                seen,
-              })
-            );
-          } catch (err) {
-            subscriber.error(err);
-            return;
-          }
+      timerSubscription = caughtSchedule(
+        subscriber,
+        scheduler,
+        () => {
```

Comment:
In the rest of places this param has been a traditional anonymous function but here an arrow function is used, is it intentional?

Reply:
Yeah. We didn't need the `this` context in this particular case. We only use `function () {}` if we need to reschedule with `this.schedule()`.